vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc

```diff
@@ -108,8 +109,7 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
           }
           vrrp_instance VI_1 {
               state MASTER
-              interface eth1
-              virtual_router_id 1
```

Comment:
why are you removing virtual_router_id ?

Reply:
Good catch. This is just a mistake. I've put this line back.